LiteCore/Database/SequenceTracker.cc

```diff
@@ -224,24 +224,22 @@ namespace litecore {
                                            uint64_t bodySize,
                                            RevisionFlags flags)
     {
+        logDebug("documentChanged('%.*s', %.*s, %llu, size=%llu, flags=%hhx",
+                 SPLAT(docID), SPLAT(revID), sequence, bodySize, flags);
         auto shortBodySize = (uint32_t)min(bodySize, (uint64_t)UINT32_MAX);
         bool listChanged = true;
         Entry *entry;
         auto i = _byDocID.find(docID);
         if (i != _byDocID.end()) {
             // Move existing entry to the end of the list:
             entry = &*i->second;
-            if (entry->isIdle() && !hasDBChangeNotifiers()) {
```

Comment:
Is `hasDBChangeNotifiers` used anywhere else?  If not can it be removed?

Reply:
Good point; the answers are No and Yes. I've removed it now.